scifin/classifier/classifier.py

```diff
@@ -119,20 +123,22 @@ def dtw_distance(ts1, ts2, window=None):
 
 # KMEANS CLUSTERING
 
-def kmeans_base_clustering(corr, names_features=None, max_num_clusters=10, n_init=10):
+@typechecked
+def kmeans_base_clustering(corr: Union[np.ndarray, pd.DataFrame], names_features: list=None,
+                           max_num_clusters: int=10, **kwargs) -> (pd.DataFrame, dict, pd.Series):
```

Comment:
just personal preference, I would prefer you break these line of codes and put the parameters on a separate new line, e.g.

```
def kmeans_base_clustering(
        corr: Union[np.ndarray, pd.DataFrame], 
        names_features: list=None,
        max_num_clusters: int=10, 
        **kwargs
) -> (pd.DataFrame, dict, pd.Series):
```

In PyCharm, you can simply auto reformat the code style by the command
https://www.jetbrains.com/help/pycharm/reformat-and-rearrange-code.html#reformat_code

Reply:
Thanks for the PyCharm tip. I've tried different breaks and did not like any, so went for an "in-between" solution. Let's keep these changes for a future cleaning.